googleapiclient/discovery.py

```diff
@@ -708,7 +708,7 @@ def createMethod(methodName, methodDesc, rootDesc, schema):
 
   parameters = ResourceMethodParameters(methodDesc)
 
-  def method(self, **kwargs):
+  def method(self, user_agent=None, **kwargs):
```

Comment:
My biggest worry with this change is that it's possible an API method will have a `user_agent` field and this will prevent that from ever being possible to set.